pandas/core/reshape/reshape.py

```diff
@@ -1096,6 +1100,162 @@ def get_empty_frame(data) -> DataFrame:
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
 
 
+def from_dummies(
```

Comment:
we should consider moving get_dummies / from_dummies to a separate file (in /reshape), could be a precursor PR.

Reply:
I like that idea to improve clarity. What would be an elegant and obvious name for a collection of "reshape operations that change the data representation" - maybe `transform`?  Or would we rather collect more categrogical/dummy specific operations instead? For me the first option seems more intuitive: I will think about a name - `/reshape/transform.py` could cause confusion with the [`.transform` method](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.transform.html)

Reply:
`one_hot_encoding.py`?

Reply:
or if its supposed to be a dummy operations file: `dummy_coding.py`?